addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -108,6 +124,53 @@ func _drop_data(at_position: Vector2, data: Variant) -> void:
 	reconnect_block.emit(block)
 
 
+func _drop_obj_property(at_position: Vector2, data: Variant) -> void:
+	var obj_name = str(data.get("object", "")).get_slice(":", 0)
+	var prop_name = data.get("property", "")
+	var val = data.get("value", null)
+	if obj_name == "" || prop_name == "" || val == null:
+		return
+
+	# Prepare a Variable block to set / get the property's value according to
+	# the modifier KEY_CTRL pressing.
+	var block_definition: BlockDefinition
+	var prop_type = typeof(val)
+	if _modifier_ctrl:
+		var type_string: String = Types.VARIANT_TYPE_TO_STRING[prop_type]
+		block_definition = (
+			BlockDefinition
+			. new(
+				&"%s_set_%s" % [obj_name, prop_name],
+				obj_name,
+				"Set the %s property" % prop_name,
+				"Variables",
+				Types.BlockType.STATEMENT,
+				prop_type,
+				"set %s to {value: %s}" % [prop_name.to_lower(), type_string],
```

Comment:
I was about to say that this is now how we show other properties previously in the catalog, but you got it right. Previously we shown them differently, and that has regressed, so I will open an issue.

Reply:
Sure.  Fixed accordingly